common/modifier_types/babylon_modifier_types.txt

```diff
@@ -368,4 +363,45 @@ country_bureaucracy_investment_cost_factor_mult = {
 	good = no
 	percent = yes
 	num_decimals = 1
+}
+
+
+###############
+##### SOL #####
+###############
```

Comment:
Wait these aren't in vanilla? seriously?

Reply:
These are, I think they were added in Cheese's Branch so I'll just delete and inform him he needs to fix it